src/applications/facility-locator/utils/facilityAddress.js

```diff
@@ -1,15 +1,20 @@
-import { compact } from 'lodash';
+import compact from 'lodash/compact';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
For what it's worth, you can use the destructuring approach or the normal `import _ from 'lodash'` and the Lodash Babel plugin will convert the imports to look like this, so it doesn't bloat the bundle more.

Reply:
Ahh, didn't know we had the babel plugin added. Last I paid attention we did not so I got in the habit of doing the direct imports (even mentioned it in my [overall review](https://gist.github.com/r-mc2/a2e269bc65eb77291de5cb5c20cef39f) of the app that I passed along months ago). 
I can definitely put it back to the destructure format if that's the case.

Reply:
I don't think we have a prescribed way of importing them, so it's up to you, I just wanted to mention that they're all equivalent from a bundle size standpoint (unless you start using `_.chain`, anyway).